class NotFoundException(Exception):
    pass


class CustomValidationError(Exception):
    pass
